CHANGES.md

```diff
@@ -35,11 +35,12 @@
 - [Breaking] Remove the `present` function and automatically switch the buffers after a draw call
 - [Breaking] Move `State` and `run` into a new `lifecycle` module
 - Add functions to determine the current framerate
-- Add the ability to customize the tick rate
+- Add the ability to customize the update and tick rate independently
```

Comment:
I choose "rate" because this PR would change the fixed timestep to a more open but controllable method, meaning there are no "ticks" anymore. It just happens as fast as possible and with a defined delay. If your calculation is fast enough between update/draw calls nothing will happen until this delay is reached, or if it is not fast enough and for example your update call takes longer than expected it'll simply catch up again as the delay is reached anyway.